### ISSUE TITLE
Difference between synchronous and async reader should be explained

### ISSUE THREAD

**Issue by marians** [nsqio/pynsq#164]
I'd like the documentation to explain what the differences between the synchronous and the async reader examples is. The might be obvious to those who developed pynsq, but aren't necessarily to somebody looking at it from the outside (like me).

Thanks!

**Comment by ploxiln** [comment]
Fair point, it could be more prominently mentioned that pynsq is for use with the tornado async framework, though it can be used in a simple synchronous mode. If you still want more explanation: http://www.tornadoweb.org/en/stable/guide/async.html

**Comment by mreiferson** [comment]
It looks like the part of the docs that @marians is referring to is in regards to sync vs. async _message handling_. That definitely could benefit from some expansion.
